src/test/java/com/zendesk/maxwell/ColumnDefTest.java

```diff
@@ -172,7 +175,7 @@ public void TestDate() {
 		assertTrue(d.matchesMysqlType(MySQLConstants.TYPE_DATE));
 
 		Date date = new GregorianCalendar(1979, 10, 1).getTime();
-		assertThat(d.toSQL(date), is("1979-10-01"));
+		assertThat(d.toSQL(date), is("''1979-11-01''"));
```

Comment:
Yes, it is returning with two single-quote characters, which I found weird, but I didn't want to change it without asking you about it first.

See https://github.com/zendesk/maxwell/blob/master/src/main/java/com/zendesk/maxwell/schema/columndef/DateColumnDef.java#L37